peps/tests/test_commands.py

```diff
@@ -33,3 +38,20 @@ def test_image_generated(self):
         img = Image.objects.get(page__path='dev/peps/pep-3001/')
         soup = BeautifulSoup(img.page.content.raw, 'lxml')
         self.assertIn(settings.MEDIA_URL, soup.find('img')['src'])
+
+    @override_settings(PEP_REPO_PATH=FAKE_PEP_REPO)
+    def test_dump_pep_pages_work(self):
+        call_command('generate_pep_pages')
+        stdout = io.StringIO()
+        sys.stdout = stdout
```

Comment:
We can replace monkeypatching with the following snippet:

```py
stdout = io.StringIO()
call_command('dump_pep_pages', stdout=stdout)
output = stdout.getvalue()
```

Reply:
👍